test/system/250-systemd.bats

```diff
@@ -292,4 +292,71 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     run_podman network rm -f $netname
 }
 
+@test "podman-play-kube@.service template" {
+    skip_if_remote "systemd units do not work with remote clients"
+
+    unit_name="podman-play-kube@.service"
+    unit_src="contrib/systemd/system/$unit_name"
+    unit_dest="$UNIT_DIR/$unit_name"
+
+    # Generate the podman-play-kube systemd template and install it
+    bin_dir=$(dirname $(realpath $PODMAN))
+    BINDIR=$bin_dir make $unit_src
+    cp $unit_src $unit_dest
```

Comment:
Why does the test have to install it? Shouldn't we expect that the test env has the correct units already installed.
I don't thing the gating test has access to the git repo (Makefile,contrib/systemd/unit)...

Reply:
You said it just before I did. This is impossible: system tests are packaged as RPMs and run in a source-less environment (gating).

Reply:
> Shouldn't we expect that the test env has the correct units already installed.

It makes local development much easier.

> I don't thing the gating test has access to the git repo (Makefile,contrib/systemd/unit)...

That brings up a valid point.  If the test installs the file(s), it would not be able to catch packaging errors (e.g., if the RHEL rpm missed installing the files).

I'll update :heavy_check_mark:

Reply:
Updated. Thanks for reviewing!